l2geth/miner/worker.go

```diff
@@ -614,6 +616,15 @@ func (w *worker) batchAnswerLoop() {
 			} else {
 				w.knowBatchPeriodAnswerMsg.Add(ev.Msg.Hash())
 			}
+			if ev.Msg.BatchIndex != w.currentBps.BatchIndex {
+				log.Error("Batch index not equal", "Current index", w.currentBps.BatchIndex, "Receive", ev.Msg.BatchIndex)
+				continue
+			}
+			if ev.Msg.StartIndex != w.eth.BlockChain().CurrentBlock().NumberU64() {
+				log.Info("Start index not equal with current height", "Current height", w.eth.BlockChain().CurrentBlock().NumberU64(), "Start index", ev.Msg.StartIndex)
+				continue
+			}
+
```

Comment:
The two checks should only be executed in scheduler node. Besides, just delete this else code:
```
			    if ev.Msg.Sequencer == w.coinbase {
					// for active sequencer
					log.Info("Active sequencer receives BatchPeriodAnswerEvent")
				} else {
					// for inactive sequencer
					log.Info("Inactive sequencer receives BatchPeriodAnswerEvent")
				}
```

Reply:
It seem like we just verify index when node is scheduler,and i want keep the sequencer logs when recive this event

Reply:
`Sequencer` doesn't need to handle `BatchPeriodAnswerEvent`. Otherwise, we can turn down the log level to debug